docker-compose.yml

```diff
@@ -0,0 +1,34 @@
+version: "3.7"
+services:
+  frozen_flame:
+    container_name: frozenFlame
+    build: .
+    image: ragingdave/FrozenFlameServer:latest
+    network_mode: "host"
```

Comment:
I think from a default docker-compose perspective, we shouldn't have the network_mode set this way. It invites port collisions, and would be better to make it configurable by using the `ports` section instead

Reply:
As you want. Comment just for people who want switch easily.